### PR TITLE
Add classification model MixNet-L

### PR DESCRIPTION
**Pull request by akorobeinikov**

### CODE REVIEW

**Comment by eaidova** [comment]
jenkins ok to test

**Comment by eizamaliev** [comment]
@alexmasterdi Has this model mean or scale values?

**Comment by eaidova** [comment]
> @alexmasterdi Has this model mean or scale values?
@eizamaliev, we know mean and scale parameters for this model, but they included in the model and our accuracy testing confirm that

File: models/public/mixnet-l/mixnet-l.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# mixnet-l
+
+## Use Case and High-Level Description
+MixNets are a family of mobile-sizes image classification models equipped with MixConv,
+a new type of mixed depthwise convolutions. There are three MixNet architectures -
+`MixNet-S` (Small), `MixNet-M` (Middle), `MixNet-L` (Large). The main differences are using
+MixConv with different kernel sizes and number of layers. Using `MixNet-L` allow to achieve greater accuracy.
```
Comment (IRDonch) [inline models/public/mixnet-l/mixnet-l.md:7]:
```suggestion
MixConv with different kernel sizes and number of layers. Using `MixNet-L` allows to achieve greater accuracy.
```

**Review by IRDonch — APPROVED** [review]
Well, the PR looks good to me. However, as far as I understand, this model is currently not supported by the IE CPU plugin, so I don't think we can merge it while that's the case. I'll keep an eye on the internal bug report; once it's fixed, we can go back to this.

File: models/public/mixnet-l/mixnet-l.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+MixNets are a family of mobile-sizes image classification models equipped with MixConv,
+a new type of mixed depthwise convolutions. There are three MixNet architectures -
+`MixNet-S` (Small), `MixNet-M` (Middle), `MixNet-L` (Large). The main differences are using
+MixConv with different kernel sizes and number of layers. Using `MixNet-L` allows to achieve greater accuracy.
+Use this [link](https://arxiv.org/abs/1907.09595) to learn more about MixNet architectures.
+`MixNet-L` was pretrained in TensorFlow\*.
+All the MixNet models have been pretrained on the ImageNet\* image database.
+For details about this family of models, check out the [TensorFlow Cloud TPU repository](https://github.com/tensorflow/tpu/tree/master/models/official/mnasnet/mixnet).
+
+## Example
```
Comment (vladimir-dudnik) [inline models/public/mixnet-l/mixnet-l.md:13]: we are removing empty section from model documentation (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)), please provide sample image or remove empty section

File: models/public/mixnet-l/mixnet-l.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+| Source framework  | TensorFlow\*  |
+
+## Accuracy
+
+| Metric | Original model | Converted model |
+| ------ | -------------- | --------------- |
+| Top 1  | 78.30%          | 78.90%           |
+| Top 5  | 93.91%          | 94.20%           |
+
+## Performance
```
Comment (vladimir-dudnik) [inline models/public/mixnet-l/mixnet-l.md:31]: please remove empty section

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]

File: models/public/mixnet-l/README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+## Use Case and High-Level Description
+MixNets are a family of mobile-sizes image classification models equipped with MixConv,
+a new type of mixed depthwise convolutions. There are three MixNet architectures -
+`MixNet-S` (Small), `MixNet-M` (Middle), `MixNet-L` (Large). The main differences are using
+MixConv with different kernel sizes and number of layers. Using `MixNet-L` allows to achieve greater accuracy.
+Use this [link](https://arxiv.org/abs/1907.09595) to learn more about MixNet architectures.
+`MixNet-L` was pretrained in TensorFlow\*.
+All the MixNet models have been pretrained on the ImageNet\* image database.
+For details about this family of models, check out the [TensorFlow Cloud TPU repository](https://github.com/tensorflow/tpu/tree/master/models/official/mnasnet/mixnet).
```
Comment (eaidova) [inline models/public/mixnet-l/README.md:11]:
```suggestion
All the MixNet models have been pretrained on the ImageNet image database.
For details about this family of models, check out the [TensorFlow Cloud TPU repository](https://github.com/tensorflow/tpu/tree/master/models/official/mnasnet/mixnet) and [paper](https://arxiv.org/abs/1907.09595).
```

File: models/public/mixnet-l/README.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+| Metric | Original model | Converted model |
+| ------ | -------------- | --------------- |
+| Top 1  | 78.30%         | 78.30%          |
+| Top 5  | 93.91%         | 93.91%          |
+
+## Input
+
+### Original Model
+
+Image, name - `image`,  shape - `[1x224x224x3]`, format is `[BxHxWxC]`, where:
```
Comment (eaidova) [inline models/public/mixnet-l/README.md:33]:
```suggestion
Image, name - `image`,  shape - `1, 224, 224, 3`, format is `B, H, W, C`, where:
```

File: models/public/mixnet-l/README.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Channel order is `RGB`.
+
+### Converted Model
+
+Image, name - `IteratorGetNext/placeholder_out_port_0`,  shape - `[1x3x224x224]`, format is `[BxCxHxW]`, where:
```
Comment (eaidova) [inline models/public/mixnet-l/README.md:44]:
```suggestion
Image, name - `IteratorGetNext/placeholder_out_port_0`,  shape - `1, 3, 224, 224]`, format is `B, C, H, W]`, where:
```

File: models/public/mixnet-l/README.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
+# mixnet-l
+
+## Use Case and High-Level Description
+MixNets are a family of mobile-sizes image classification models equipped with MixConv,
+a new type of mixed depthwise convolutions. There are three MixNet architectures -
+`MixNet-S` (Small), `MixNet-M` (Middle), `MixNet-L` (Large). The main differences are using
+MixConv with different kernel sizes and number of layers. Using `MixNet-L` allows to achieve greater accuracy.
+Use this [link](https://arxiv.org/abs/1907.09595) to learn more about MixNet architectures.
```
Comment (vladimir-dudnik) [inline models/public/mixnet-l/README.md:8]:
```suggestion
Refer to original [article](https://arxiv.org/abs/1907.09595) to learn more about MixNet architectures.
```

File: models/public/mixnet-l/README.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+`MixNet-L` was pretrained in TensorFlow\*.
```
Comment (vladimir-dudnik) [inline models/public/mixnet-l/README.md:9]:
```suggestion
`MixNet-L` was pretrained in TensorFlow\* with ImageNet\* dataset.
```

File: models/public/mixnet-l/README.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+All the MixNet models have been pretrained on the ImageNet\* image database.
```
Comment (vladimir-dudnik) [inline models/public/mixnet-l/README.md:10]:
```suggestion
```

File: models/public/mixnet-l/README.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+- `B` - batch size
+- `C` - predicted logits for each class
+
+### Converted Model
+
+Object classifier according to ImageNet classes, name - `logits`,  shape - `1,1000`, output data format is `B,C` where:
+
+- `B` - batch size
+- `C` - predicted logits for each class
+
```
Comment (vladimir-dudnik) [inline models/public/mixnet-l/README.md:68]: add new common to all models section, before Legal Information

`
## Download a Model and Convert it into Inference Engine Format

You can download models and if necessary convert them into Inference Engine format using the [Model Downloader and other automation tools](../../../tools/downloader/README.md) as shown in the examples below.

An example of using the Model Downloader:
```
python3 <omz_dir>/tools/downloader/downloader.py --name <model_name>
```

An example of using the Model Converter:
```
python3 <omz_dir>/tools/downloader/converter.py --name <model_name>
```
`

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]
lease also add model to test cases

File: models/public/mixnet-l/README.md (hunk 10 above)
Comment (akorobeinikov) [reply models/public/mixnet-l/README.md:66]: added

File: models/public/mixnet-l/README.md (hunk 7 above)
Comment (akorobeinikov) [reply models/public/mixnet-l/README.md]: Please see new version of `Use Case and High-Level Description`. Perhaps it is a better variant (suggested Aidova Ekaterina).

File: models/public/mixnet-l/README.md [hunk 11]
```diff
@@ -0,0 +1,85 @@
+# mixnet-l
+
+## Use Case and High-Level Description
+MixNets are a family of mobile-sizes image classification models equipped with MixConv,
+a new type of mixed depthwise convolutions. There are three MixNet architectures -
+`MixNet-S` (Small), `MixNet-M` (Middle), `MixNet-L` (Large). The main differences are using
+MixConv with different kernel sizes and number of layers. Using `MixNet-L` allows to achieve greater accuracy.
+All the MixNet models have been pretrained on the ImageNet image database.
```
Comment (vladimir-dudnik) [inline models/public/mixnet-l/README.md:8]:
```suggestion
All the MixNet models have been pretrained on the ImageNet dataset.
```

File: models/public/mixnet-l/README.md (hunk 7 above)
Comment (vladimir-dudnik) [reply models/public/mixnet-l/README.md]: ok with small correction. Seems ImageNet dataset looks more appropriate than ImageNet image database

**Review by vladimir-dudnik — APPROVED** [review]

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build
